packages/kube-client/lib/resources/GardenerCore.js

```diff
@@ -66,6 +66,10 @@ class Shoot extends mix(GardenerCore).with(NamespaceScoped, Readable, Observable
     return this.create([namespace, name, 'adminkubeconfig'], body, options)
   }
 
+  patchBindingResource (namespace, name, body, options) {
```

Comment:
rename to `updateBinding` according to https://github.com/gardener/gardener/blob/master/pkg/apis/core/v1beta1/types_shoot.go#L31 or if we stick to our naming convention `mergePatchBinding`.

Reply:
function removed.